src/lib/github/gh-utility.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * function that returns common payload properties.
+ */
+
+import { Context } from "./types";
+
+export default function commonPayloadProps (context: Context): any {
```

Comment:
Please use a more concise exact return type. `any` is very general.

Rename function to `getBasicRepoProps`